nilearn/plotting/displays/_slicers.py

```diff
@@ -219,7 +219,8 @@ def title(self, text, x=0.01, y=0.99, size=15, color=None, bgcolor=None,
 
     @fill_doc
     def add_overlay(self, img, threshold=1e-6, colorbar=False,
-                    cbar_tick_format="%.2g", **kwargs):
+                    cbar_tick_format="%.2g", cbar_vmin=None, cbar_vmax=None,
```

Comment:
I just realized it could be worth adding a docstring for these new parameters as the others are documented. WDYT?